robottelo/ui/locators.py

```diff
@@ -672,6 +672,13 @@ def __iter__(self):
     "entity_checkbox": (
         By.XPATH,
         "//label[normalize-space(.)='%s']/input[@type='checkbox']"),
+    "entity_select_list": (
+        By.XPATH,
+        "//ul/li/div[normalize-space(.)='%s']"),
```

Comment:
Let's me continue with other entities and correct that line in case I face any issue. I think there are equal chances to get wrong element or to get `@class='something there that is not select2-results'`